### PR TITLE
Fixed redirect errors (301)  on  3  domains

### PR DESCRIPTION
**Pull request by najela**
This fixes the redirect errors on the links below such that 

http://twitter.com*   becomes **https://**twitter.com*
http://www.packtpub.com*  becomes  **https://**www.packtpub.com*
http://www.flickr.com* becomes **https://**www.flickr.com**

This PR is meant to fix all the [present errors reported](http://jenkins.jquery.com/job/code.jquery.com/lastBuild/console) on the latest build for the http://code.jquery.com/ website

### CODE REVIEW

**Comment by gnarf** [comment]
I'm :+1: here - thanks for doing this!

**Comment by arschmitz** [comment]
@najela huge thanks for working on all these. Making all the time i put into making spider.js worth it!

**Comment by najela** [comment]
@arschmitz  I will actually like to see the code for the crawler which does this checks. Please where is it found?

**Comment by arthurvr** [comment]
Thanks for doing this! The code for spider lives in this repo: https://github.com/arschmitz/spider.js. I'm on mobile at the moment but will merge once I get a chance. Thanks!

**Comment by arschmitz** [comment]
@arthurvr Thanks for posting the link just saw this. @najela there is also a grunt task for it https://github.com/arschmitz/grunt-spider ( though i need to update the documentation for it to match latest spider.js )

File: themes/jquery/css/font-awesome.css [hunk 1]
```diff
@@ -17,7 +17,7 @@
  *  Contact
  *  -------------------------------------------------------
  *  Email: dave@davegandy.com
- *  Twitter: http://twitter.com/fortaweso_me
+ *  Twitter: https://twitter.com/fortaweso_me
```
Comment (arthurvr) [inline themes/jquery/css/font-awesome.css]: We shouldn't be changing vendor files IMHO

File: themes/learn.jquery.com/content-page.php [hunk 2]
```diff
@@ -20,7 +20,7 @@
 		<div class="bottom-links row">
 			<?php if (isset($next_prev_arr['prev'])): ?>
 				<div class="prev six columns">
-					<a href="<?php echo $next_prev_arr['prev']->guid; ?>">
+					<a href="<?php echo get_page_link( $next_prev_arr['prev']->ID ); ?>">
```
Comment (arthurvr) [inline themes/learn.jquery.com/content-page.php]: This fixes https://github.com/jquery/learn.jquery.com/issues/647, right?

Comment (arthurvr) [inline themes/learn.jquery.com/content-page.php]: Could you remove these changes? We already had an open PR for that: https://github.com/jquery/jquery-wp-content/pull/358. I feel like we should attribute @sigorilla in here as [he was the one that fixed it](https://github.com/jquery/learn.jquery.com/issues/647#issuecomment-90430039).

Comment (najela) [inline themes/learn.jquery.com/content-page.php]: Yep I'll remove the changes. I was actually trying to test @sigorilla's changes forgetting that my commit will be attached to this PR

File: themes/jquery/css/font-awesome-ie7.min.css [hunk 3]
```diff
@@ -17,6 +17,6 @@
  *  Contact
  *  -------------------------------------------------------
  *  Email: dave@davegandy.com
- *  Twitter: http://twitter.com/fortaweso_me
+ *  Twitter: https://twitter.com/fortaweso_me
```
Comment (arthurvr) [inline themes/jquery/css/font-awesome-ie7.min.css]: Same thing here. I wonder if we want to change vendor files.

**Comment by najela** [comment]
9567adf Solves the raised issues. I now understand why submitting a PR from your master branch is a bad idea :)

File: themes/jquery/css/font-awesome.css (hunk 1 above)
Comment (najela) [inline themes/jquery/css/font-awesome.css]: I buy your idea :100:/:100:. Such errors will also not get noticed by jenkins

**Comment by arschmitz** [comment]
@najela you don't need to worry about jenkins for errors in the 3rd party files we can configure ignore params for any of the jobs to fix  things like this. Just let me know what ever needs to be ignored and ill configure it.

**Pull request closed without merging**